packages/components/Menu/src/MenuList/MenuList.tsx

```diff
@@ -89,7 +89,6 @@ export const MenuList = compose<MenuListType>({
         Platform.OS === 'macos' ? (
           <Slots.root onMouseLeave={setFocusZoneFocus} onKeyDown={menuList.onListKeyDown}>
             <Slots.scrollView
-              accessibilityRole="menu"
```

Comment:
Should we gate this for win32? Some accessibilityRoles are only available on netui, so this might be intended for win32

Reply:
Nvm there's already a platform check!

Reply:
Talked offline - this block of code should only execute on macOS.